src/cocotb/handle.py

```diff
@@ -1235,6 +1265,12 @@ def value(self) -> bytes:
     def value(self, value: bytes) -> None:
         self.set(value)
 
+    @deprecated(
+        "`str(handle)` casts have been deprecated. Use `handle.value.decode('ascii')` instead."
```

Comment:
```suggestion
        '`str(handle)` casts have been deprecated. Use `handle.value.decode("ascii")` instead.'
```
Let's suggest code that `black` won't change for users.